project-proposals/stage-1/Trainpositioning.md

```diff
@@ -0,0 +1,125 @@
+
+## Which exact repositories do you intend to transfer to the GitHub organization of the OpenRail Association?
+
+Originally, the repository contains a JSON API. We will split of the logic from the API and put it in a separate repository. The API wrapper is not very useful to opensource, so we will only opensource the library repository.
+
+
+## What is the project's main license?
+
+Apache License 2.0
+
+## What other licenses does the project use, e.g., for included 3rd-party code?
+
+
+We are still in the progress of simplifying the library, but at the moment, these packages are used with their respective licenses:
+
+- NetTopologySuite: BSD-3-Clause License
+- NodaTime: Apache License 2.0
+- Npgsql.NetTopologySuite: PostgreSQL License
+- Serilog: Apache License 2.0
+- Serilog.Sinks.Console: Apache License 2.0
+- DotSpatial.Projections: MIT License
+
+We don't immediately see any major issues.
+
+## Are any trademarks associated with the project?
+
+No
+
+## Does the project have a web site? Where is it? Are you ok with moving it to be hosted by the OpenRail Association?
+
+No, no website. We will foresee documentation and a separate .net project to run tests, both will also be open sourced.
+
+
+## What are the communication channels the project uses (such as mailing lists, Slack, IRC, etc.)?
+
+Git issues
+
+## What is the project's leadership team and decision-making process?
+
+- PM: glenn.vancalster@infrabel.be
+- Dev: kevin.goos@infrabel.be
+- Dev: ilya.quateau@infrabel.be
+- Prev PM / Dev: mathias.vandenauweele@infrabel.be
+
+## How is it decided if and when a pull request is merged?
+
+Depends on the pull request, generally the project leader decides.
+
+## How can someone become a committer or a maintainer to/of the project?
+
+After discussion with the project leader and current development team.
+
+## How is development of the project planned and organized? Is this transparent to the public?
+
+Using git issues & epics/stories.
+
+## What is the project's roadmap?
+
+After the project refactoring and extensive testing, no further development is planned.
+
+## What other organizations in the world should be interested in this project?
+
+All organisations that use measurement trains and process the measurements. But the project can also be used as a guideline for organisations that want to post processes location measurements for other domains that include topology and navigability.
+
+## Why would this project be a good candidate for inclusion in the OpenRail Association?
+
+Because all infra managers do this and should better do it together.
+
+## Are there competing products or projects? If there are, please explain how the proposed projects differentiates.
+
+
+Not that we know of.
+
+## What standards does the project implement or rely on? How are they related to other existing standards?
+
+Topology and navigability is also defined by several other railway related standards. But they are very loosely related.

```

Comment:
Yes, please review the presentation given on the BoD

![image](https://github.com/user-attachments/assets/43cb7da7-ec29-47ee-8f96-c1c8c60de1a2)

![image](https://github.com/user-attachments/assets/22a218ea-8517-44c1-85e3-09827f467078)

Reply:
Not all members of the Technical Committee have access to the BoD materials. So if there is any more info about the context and how to understand the diagrams please add them here as well.

Reply:
RCM-DX is a data format. The RCM-DX file can contain GNSS data. In case topological locations are also requested, TP-Lib could be used to read data out of the file and write its result back into the RCM-DX file. TP-lib is currently not able to read/write to RCM-DX but an implementation could be developed.